tests/src/Factory/Api/Twitter/ActvitiesTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Friendica\Test\src\Factory\Api\Twitter;
+
+use Friendica\DI;
+use Friendica\Factory\Api\Friendica\Activities;
+use Friendica\Test\FixtureTest;
+
+class ActvitiesTest extends FixtureTest
```

Comment:
Typo in class/file names:
```suggestion
class ActivitiesTest extends FixtureTest
```